src/itty-router.d.ts

```diff
@@ -16,19 +18,6 @@ export interface RouteEntry<TRequest> {
   2: RouteHandler<TRequest>[]
 }
 
-export interface Request {
-  method: string
-  params?: Obj
-  query?: Obj
```

Comment:
I left a comment as well, but this is an itty-specific property not covered by standard Request